tor/tor.go

```diff
@@ -220,12 +220,20 @@ func ResolveTCPAddr(address, socksAddr string) (*net.TCPAddr, error) {
 		return nil, err
 	}
 
-	ip, err := LookupHost(host, socksAddr)
+	p, err := strconv.Atoi(port)
 	if err != nil {
 		return nil, err
 	}
 
-	p, err := strconv.Atoi(port)
+	// Do we already have an IP? Then we don't need to look up anything.
```

Comment:
Let's update the docs too since we are actually using `net` now. And maybe a `TODO` or `NOTE` mentioning that this is a temp fix.

Reply:
IMO this isn't a temp fix, just that we don't need to try to resolve things if we _already_ have an IP addr. The PR linked above in `btcd` fixes somethign different: to support parsing out a response for name resolution that gives us an IPv6 addr.